jsf-api/src/main/resources/jsf.js

```diff
@@ -2909,6 +2909,10 @@ if (!((jsf && jsf.specversion && jsf.specversion >= 20000 ) &&
                         responseType = partialResponse.childNodes[i];
                         break;
                     }
+
+                    if (partialResponse.childNodes[i].nodeName === "redirect") {
```

Comment:
@vsingleton, @manfredriem, @edburns,
The "Receiving The Ajax Response" section of the JSF Specification (section 13.3.4) states that:

> The elements in the response must be processed in the order they appear in the response.

However, [lines 2817-2822 of **`jsf.js`**](https://github.com/javaserverfaces/mojarra/blob/2.2.13/jsf-api/src/main/resources/jsf.js#L2817-L2822) seem to process `<error>` elements before any other elements. If this is not a violation of the Spec, then `<redirect>` elements should recieve similar treatment, and this fix is valid. What do you think?